src/index.html

```diff
@@ -135,6 +135,26 @@
       display: none !important;
     }
   </style>
+
+  <!-- AppD EUM -->
+  <script charset='UTF-8'>
+    var hostname = window.location.hostname,
+      nonProdHosts = ['library.artstor.org', 'localhost', '127.0.0.1'],
+      prodKey = 'AD-AAB-AAM-TVS',
+      testKey = 'AD-AAB-AAM-TWZ';
+
+    window['adrum-start-time'] = new Date().getTime();
+    (function(config){
+      config.appKey = nonProdHosts.includes(hostname) ? testKey : prodKey;
```

Comment:
This is the one piece of custom logic I added to the out of the box script provided by appD. It essentially allows us to report to test/prod. I ended up using the `hostname` as criteria for determining test/prod, but let me know if there is a better way to get at the environment info!

FWIW running locally I am starting to start seeing some metric come in for the TEST 🎉  https://ithaka.saas.appdynamics.com/controller/#/location=EUM_WEB_MAIN_DASHBOARD&timeRange=last_1_hour.BEFORE_NOW.-1.-1.60&application=1070